packages/core/package.json

```diff
@@ -233,6 +233,85 @@
                     "type": "object",
                     "description": "%AWS.configuration.description.lambda.recentlyUploaded%",
                     "default": []
+                },
+                "aws.amazonq.logLevel": {
```

Comment:
If amazonq commands are declared here in core package.json, is it OK to remove the amazonq command in /amazonq/package.json ?

Reply:
If they are duplicated in both q and core, when I change Q settings, should I update both duplicate or ?

Reply:
@leigaol see `docs/arch_develop.md`: https://github.com/aws/aws-toolkit-vscode/pull/4669/commits/964cc471f2a91e20ce94052c9638f8cd5b6495bb